apps/shell/tash_main.c

```diff
@@ -156,9 +167,18 @@ char *tash_read_input_line(int fd)
 				shdbg("TASH: can not read uart\n");
 				return buffer;
 			}
-
+#ifdef CONFIG_PM
+			pmdrv_fd = open(PM_DRIVER, O_WRONLY);
```

Comment:
We dont need to open the pm device for each character. We can open it in tash_init and keep it open as long as tash is alive.

Reply:
I have created the new `tash_pm_timedsuspend` API

Reply:
@gSahitya-samsung Could you check @kishore-sn 's comment again? Changed code also opens and closes the pm device at every character.

Reply:
Done